wx.lib.agw.buttonpanel broken in linux gtk3. Fix included.

Running on Linux debian buster up to date. (gtk3 python 3.7)
wxpython self compiled from source on master branch

The sample code for wx.lib.agw.buttonpanel as posted at https://wxpython.org/Phoenix/docs/html/wx.lib.agw.buttonpanel.html?highlight=buttonpanel#usage crashes.

This is the code in question:
``` python
import wx
import wx.lib.agw.buttonpanel as BP

class MyFrame(wx.Frame):

    def __init__(self, parent, id=-1, title="ButtonPanel", pos=wx.DefaultPosition,
                 size=(800, 600), style=wx.DEFAULT_FRAME_STYLE):

        wx.Frame.__init__(self, parent, id, title, pos, size, style)

        mainPanel = wx.Panel(self, -1)
        self.logtext = wx.TextCtrl(mainPanel, -1, "", style=wx.TE_MULTILINE)

        vSizer = wx.BoxSizer(wx.VERTICAL)
        mainPanel.SetSizer(vSizer)

        titleBar = BP.ButtonPanel(mainPanel, -1, "A Simple Test & Demo")

        btn1 = BP.ButtonInfo(titleBar, wx.ID_ANY, wx.Bitmap("png4.png", wx.BITMAP_TYPE_PNG))
        titleBar.AddButton(btn1)
        self.Bind(wx.EVT_BUTTON, self.OnButton, btn1)

        btn2 = BP.ButtonInfo(titleBar, wx.ID_ANY, wx.Bitmap("png3.png", wx.BITMAP_TYPE_PNG))
        titleBar.AddButton(btn2)
        self.Bind(wx.EVT_BUTTON, self.OnButton, btn2)

        btn3 = BP.ButtonInfo(titleBar, wx.ID_ANY, wx.Bitmap("png2.png", wx.BITMAP_TYPE_PNG))
        titleBar.AddButton(btn3)
        self.Bind(wx.EVT_BUTTON, self.OnButton, btn3)

        btn4 = BP.ButtonInfo(titleBar, wx.ID_ANY, wx.Bitmap("png1.png", wx.BITMAP_TYPE_PNG))
        titleBar.AddButton(btn4)
        self.Bind(wx.EVT_BUTTON, self.OnButton, btn4)

        vSizer.Add(titleBar, 0, wx.EXPAND)
        vSizer.Add((20, 20))
        vSizer.Add(self.logtext, 1, wx.EXPAND | wx.ALL, 5)

        titleBar.DoLayout()
        vSizer.Layout()


    def OnButton(self, event):
        ''' Handler for the ``wx.EVT_BUTTON`` event. '''

        obj = event.GetEventObject()

        # This will print the button label
        print(obj.GetText())


# our normal wxApp-derived class, as usual

app = wx.App(0)

frame = MyFrame(None)
app.SetTopWindow(frame)
frame.Show()

app.MainLoop()
```

and this is the complete error log written to the console:

```
Traceback (most recent call last):
  File "test.py", line 56, in <module>
    frame = MyFrame(None)
  File "test.py", line 20, in __init__
    titleBar.AddButton(btn1)
  File "/usr/local/lib/python3.7/dist-packages/wx/lib/agw/buttonpanel.py", line 1934, in AddButton
    self._mainsizer.Insert(lenChildren-1, btnInfo, 0, wx.ALIGN_CENTER|wx.EXPAND, userData=btnInfo)
  File "/usr/local/lib/python3.7/dist-packages/wx/lib/agw/buttonpanel.py", line 1161, in Insert
    szitem = wx.BoxSizer.Insert(self, before, sz, proportion, flag, border, item)
wx._core.wxAssertionError: C++ assertion "!(flags & (wxALIGN_BOTTOM | wxALIGN_CENTRE_VERTICAL))" failed at /home/jorge/vc/Phoenix/ext/wxWidgets/src/common/sizer.cpp(2108) in DoInsert(): Vertical alignment flags are ignored with wxEXPAND
```

**FIX**
It is sufficient to remove `|wx.EXPAND` from line 1934 of buttonpanel.py for the code above to run and work correctly.

--- a/demo/PDFViewer.py
+++ b/demo/PDFViewer.py
@@ -17,15 +17,14 @@ class TestPanel(wx.Panel):
         vsizer = wx.BoxSizer( wx.VERTICAL )
         self.buttonpanel = pdfButtonPanel(self, wx.ID_ANY,
                                 wx.DefaultPosition, wx.DefaultSize, 0)
-        vsizer.Add(self.buttonpanel, 0,
-                                wx.GROW|wx.ALIGN_CENTER_VERTICAL|wx.LEFT|wx.RIGHT|wx.TOP, 5)
+        vsizer.Add(self.buttonpanel, 0, wx.GROW|wx.LEFT|wx.RIGHT|wx.TOP, 5)
         self.viewer = pdfViewer( self, wx.ID_ANY, wx.DefaultPosition,
                                 wx.DefaultSize, wx.HSCROLL|wx.VSCROLL|wx.SUNKEN_BORDER)
         vsizer.Add(self.viewer, 1, wx.GROW|wx.LEFT|wx.RIGHT|wx.BOTTOM, 5)
         loadbutton = wx.Button(self, wx.ID_ANY, "Load PDF file",
                                 wx.DefaultPosition, wx.DefaultSize, 0 )
         vsizer.Add(loadbutton, 0, wx.ALIGN_CENTER|wx.ALL, 5)
-        hsizer.Add(vsizer, 1, wx.GROW|wx.ALIGN_CENTER_HORIZONTAL|wx.ALL, 5)
+        hsizer.Add(vsizer, 1, wx.GROW|wx.ALL, 5)
         self.SetSizer(hsizer)
         self.SetAutoLayout(True)
 

--- a/wx/lib/agw/buttonpanel.py
+++ b/wx/lib/agw/buttonpanel.py
@@ -1931,7 +1931,7 @@ class ButtonPanel(wx.Panel):
         """
 
         lenChildren = len(self._mainsizer.GetChildren())
-        self._mainsizer.Insert(lenChildren-1, btnInfo, 0, wx.ALIGN_CENTER|wx.EXPAND, userData=btnInfo)
+        self._mainsizer.Insert(lenChildren-1, btnInfo, 0, wx.ALIGN_CENTER, userData=btnInfo)
 
         self._vButtons.append(btnInfo)
 

--- a/wx/lib/pdfviewer/viewer.py
+++ b/wx/lib/pdfviewer/viewer.py
@@ -145,6 +145,7 @@ class pdfViewer(wx.ScrolledWindow):
         Redraw on resize.
         """
         if self.resizing:
+            self.page_buffer_valid = False
             self.Render()
             self.resizing = False
         event.Skip()

